srcpkgs/font-adobe-source-code-pro/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'font-adobe-source-code-pro'
 pkgname=font-adobe-source-code-pro
-version=2.030R
+version=1.010
 revision=1
-wrksrc="source-code-pro-${version}-ro-1.050R-it"
 archs=noarch
+build_style=fetch
 depends="font-util"
-font_dirs="/usr/share/fonts/OTF/"
 short_desc="Monospaced font family for user interface and coding environments"
 maintainer="Piero La Terza <platerza@protonmail.com>"
 license="OFL-1.1"
 homepage="https://adobe-fonts.github.io/source-code-pro/"
-distfiles="https://github.com/adobe-fonts/source-code-pro/archive/${version}-ro/1.050R-it.tar.gz"
-checksum=a4e4dd59b8e0a436b934f0f612c2e91b5932910c6d1c3b7d1a5a9f389c86302b
+distfiles="https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Italic.otf
```

Comment:
need it to be versioned.

Reply:
`variable-fonts` is the release tag. Annoying, I know. See [here](https://github.com/adobe-fonts/source-code-pro/tags).

Let me know how you'd prefer this to be formatted.

Reply:
I don't know if they can be properly checked for updates so please take some time to check for updates manually for it.